tiledb/sm/query/query.cc

```diff
@@ -403,6 +403,12 @@ std::vector<std::string> Query::buffer_names() const {
     ret.push_back(constants::coords);
   }
 
+  if (uses_dimension_labels()) {
```

Comment:
This function is called by other places of the code... If we need to add this only for serialization, I'd like to make a separate function.

Reply:
Done